src/cprover/help_formatter.h

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************\
+
+Module: Help Formatter
+
+Author: Daniel Kroening, dkr@amazon.com
+
+\*******************************************************************/
+
+/// \file
+/// Help Formatter
+
+#ifndef CPROVER_CPROVER_HELP_FORMATTER_H
+#define CPROVER_CPROVER_HELP_FORMATTER_H
```

Comment:
How does this differ from `help_entry`, and why is it part of the `cprover` tool (only)? I can see that the implementation does handle help strings with some markup, but can't we please instead have this for everyone rather than having it buried in some commit? I had put together #6931, which perhaps could be done in a more human-readable fashion using this facility.